features/storage/kvstore/tdbstore/TDBStore.cpp

```diff
@@ -24,7 +24,10 @@
 #include "mbed_error.h"
 #include "mbed_wait_api.h"
 #include "MbedCRC.h"
+//Bypass the check of NVStore co existance if compiled for TARGET_TFM
+#if !(TARGET_TFM_BYPASS_NVSTORE_CHECK)
```

Comment:
Please change the define from TARGET_TFM_BYPASS_NVSTORE_CHECK to TARGET_BYPASS_NVSTORE_CHECK.
The motivation is good for the comment but not required in the name

Reply:
Done